src/reducers/user.js

```diff
@@ -15,14 +15,20 @@ const SET_PREFIX = 'user/SET_PREFIX';
 const SET_COMPLETED = 'user/SET_COMPLETED';
 const SET_TRACKING_ID = 'user/SET_TRACKING_ID';
 
+// Provide the referrer based on whatever our ref is set to at page load.
+const referrerMatch = window.location.search.match(/\?ref=([a-zA-Z*])/);
```

Comment:
wasn't this added in another pr?

Reply:
yes; this PR is reliant on those changes and they're included in this branch in commit https://github.com/steemit/faucet/pull/366/commits/8466046f5729bcec77997fb626dd3e63a926c623